lib/dotcom_web/components/trip_planner/itinerary_summary.ex

```diff
@@ -99,11 +109,15 @@ defmodule DotcomWeb.Components.TripPlanner.ItinerarySummary do
       <%= for {route, index} <- Enum.with_index(@routes) do %>
         <.route_symbol route={route} class={"#{@grouped_classes} #{zindex(index)} #{@class}"} />
 
-        <div
-          :if={@slashed? and index < Kernel.length(@routes) - 1}
-          class={"bg-white -mt-0.5 w-1 h-7 #{zindex(index)} transform rotate-[17deg]"}
-        >
-        </div>
+        <%= if @slashed? and index < Kernel.length(@routes) - 1 do %>
+          <div
+            class={"bg-white -mt-0.5 w-1 h-7 #{zindex(index)} transform rotate-[17deg]"}
+            aria-label="or"
```

Comment:
📓 "Or" in between combined routes

Reply:
This `"or"` only shows up for combined bus routes, not combined subway routes.